Lib/test/test_urllibnet.py

```diff
@@ -213,14 +214,44 @@ def test_context_argument(self):
         self.assertIn("Python", response.read())
 
 
+class urlopen_FTPTest(unittest.TestCase):
+    # TODO: https://github.com/python/pythondotorg/issues/1069
+    # Use a resource hosted at python.org
+    FTP_TEST_FILE = 'ftp://ftp.debian.org/debian/README'
+
+    def test_multiple_ftp_retrieves(self):
+
+        with test_support.transient_internet(self.FTP_TEST_FILE):
+            try:
+                for _ in range(3):
```

Comment:
`for _ in range(3):` is used a couple of times in this class. I think the 3 should be replaced with a constant.